openslides/motions/views.py

```diff
@@ -87,12 +87,25 @@ def create(self, request, *args, **kwargs):
         """
         # Check permission to send some data.
         if not has_perm(request.user, 'motions.can_manage'):
-            whitelist = (
+            whitelist = [
                 'title',
                 'text',
                 'reason',
                 'comments',  # This is checked later.
-            )
+            ]
+            if 'parent_id' in request.data:  # For creating amendments.
+                try:
+                    parent_motion = Motion.objects.get(pk=request.data['parent_id'])
+                except Motion.DoesNotExist:
+                    raise ValidationError({'detail': _('The parent motion does not exist.')})
+                else:
+                    whitelist.extend([
+                        'parent_id',
+                        'category_id',      # This will be set to the matching
+                        'motion_block_id',  # values from parent_motion.
+                    ])
+                    request.data['category_id'] = parent_motion.category.id
+                    request.data['motion_block_id'] = parent_motion.motion_block.id
             for key in request.data.keys():
                 if key not in whitelist:
                     # Non-staff users are allowed to send only some data.
```

Comment:
@emanuelschuetze can you put here a `print("{} not in whitelist".format(key))`? This may give us the wrong key.